src/components/v5/common/ActionSidebar/partials/VerifiedMembersSelect/VerifiedMembersSelect.tsx

```diff
@@ -0,0 +1,184 @@
+import clsx from 'clsx';
+import { utils } from 'ethers';
+import React, { FC } from 'react';
+import { useController, useFormContext, useWatch } from 'react-hook-form';
+
+import { useAdditionalFormOptionsContext } from '~context/AdditionalFormOptionsContext/AdditionalFormOptionsContext';
+import {
+  useRelativePortalElement,
+  useUserByAddress,
+  useToggle,
+  useMobile,
+} from '~hooks';
+import Icon from '~shared/Icon';
+import { formatText } from '~utils/intl';
+import Button from '~v5/shared/Button';
+import SearchSelect from '~v5/shared/SearchSelect/SearchSelect';
+import UserAvatar from '~v5/shared/UserAvatar';
+
+import { AddRemoveMembersType } from '../forms/ManageVerifiedMembersForm/consts';
+
+import { useVerifiedMembersSelect } from './hooks';
+import { VerifiedMembersSelectProps } from './types';
+
+const displayName = 'v5.common.ActionsContent.partials.VerifiedMembersSelect';
+
+const VerifiedMembersSelect: FC<VerifiedMembersSelectProps> = ({
+  name,
+  fieldArrayMethods,
+}) => {
+  const {
+    field,
+    fieldState: { error },
+  } = useController({
+    name,
+  });
+  const isError = !!error;
+  const addRemoveMembers: AddRemoveMembersType | undefined = useWatch({
+    name: 'addRemoveMember',
+  });
+  const { usersOptions } = useVerifiedMembersSelect();
+  const [
+    isUserSelectVisible,
+    {
+      toggle: toggleUserSelect,
+      toggleOff: toggleOffUserSelect,
+      registerContainerRef,
+    },
+  ] = useToggle();
+  const { user: userByAddress } = useUserByAddress(field.value.value);
+  const { readonly } = useAdditionalFormOptionsContext();
+  const isMobile = useMobile();
+
+  const { portalElementRef, relativeElementRef } = useRelativePortalElement<
+    HTMLButtonElement,
+    HTMLDivElement
+  >([isUserSelectVisible]);
+  const showOnlyNotVerifiedMembers =
+    addRemoveMembers === AddRemoveMembersType.Add;
+  const { getValues, setValue } = useFormContext();
```

Comment:
I would decouple this from all form related things, pass in the data from the form, it's cleaner that way